src/deno/stable/variables/resources.ts

```diff
@@ -0,0 +1,12 @@
+///<reference path="../lib.deno.d.ts" />
+
+export const resources: typeof Deno.resources = function resources() {
+  console.warn(
+    [
+      "Deno.resources() shim returns a dummy object that does not update.",
+      "If you think this is a mistake, raise an issue at https://github.com/fromdeno/deno.ns/issues",
+    ].join("\n"),
+  );
+
+  return {};
```

Comment:
```suggestion
  return { "0": "stdin", "1": "stdout", "2": "stderr" };
```

Reply:
I would advise against it. Sure by default you got those 3 resources, but they might be closed. I see no point in shimming anything here, as `Deno.resources()` is used only in testing and gives some overview of what's open in the runtime, but since there's no equivalent in Node then I wouldn't sweat it.

Reply:
> `Deno.resources()` is used only in testing

Why isn't it deprecated then? Test runner is now in Rust, isn't it?